src/client.jsx

```diff
@@ -73,6 +76,16 @@ class SockJsClient extends React.Component {
   _initStompClient = () => {
     // Websocket held by stompjs can be opened only once
     this.client = Stomp.over(new SockJS(this.props.url));
+    if (this.props.heartbeat) {
+      this.client.heartbeat.outgoing = this.props.heartbeat;
+      this.client.heartbeat.incoming = this.props.heartbeat;
+    }
+    if (Object.keys(this.props).includes('heartbeatIncoming')) {
```

Comment:
@wedgemartin Let's define the default values in `defaultProps` and remove the if else entirely. We can use the defaults provided by stomjs for it (10000ms)